packages/notebook/src/actions.tsx

```diff
@@ -549,7 +549,10 @@ export namespace NotebookActions {
       translator
     );
 
-    void Private.handleRunState(notebook, state, false);
+    // While we do not want to scroll if cell is visible,
+    // we still need to invoke the scrolling logic
+    // in case if cell is outside of viewport.
```

Comment:
I generally lean toward more comments than fewer, but in this case I think it would better to remove the comment or move it to the `handleRunState` function definition.

```suggestion
```